libraries/adaptive-expressions/src/expressionEvaluator.ts

```diff
@@ -5,21 +5,21 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { Expression, ReturnType } from './expression';
+import { ReturnType } from './returnType';
 import { MemoryInterface } from './memory';
 import { Options } from './options';
 
 /**
  * Delegate for doing static validation on an expression.
  * Validators can and should throw exceptions if the expression is not valid.
  */
-export type ValidateExpressionDelegate = (expression: Expression) => any;
+export type ValidateExpressionDelegate = (expression: any) => any;
```

Comment:
>any [](start = 54, length = 3)

Why are these no longer Expression?  We have lost any type safety and yet this must be an Expression, right?